src/components/menubar/menubar.tsx

```diff
@@ -483,62 +500,67 @@ export class ZanitMenubar {
-                          href={subitem.href}
-                          id={subitem.id}
-                          role="menuitem"
-                          tabIndex={-1}
-                          aria-haspopup={subitem.menuItems?.length ? 'true' : 'false'}
-                          aria-expanded={
-                            subitem.menuItems?.length ? (this.openMenu === subitem.id ? 'true' : 'false') : undefined
-                          }
-                          aria-current={this.current === item.id ? 'page' : 'false'}
-                          onPointerOver={() => this.showMenu(subitem)}
-                          onKeyDown={(event) => this.handleItemKeydown(event, subitem)}
-                        >
-                          <span>{subitem.label}</span>
-                          {subitem.menuItems?.length > 0 && (
-                            <z-icon
-                              name={this.openMenu === subitem.id ? 'chevron-up' : 'chevron-down'}
-                              width="0.75rem"
-                              height="0.75rem"
-                            />
-                          )}
-                        </a>
-                      </li>
-                    </Fragment>
-                  ))}
-                </ul>
-                {item.navbarItems.map(
-                  (subitem) =>
-                    this.openMenu === subitem.id && (
-                      <Menu
-                        controlledBy={subitem.id}
-                        items={subitem.menuItems}
-                        current={this.current}
-                        onItemKeyDown={(event) => this.handleMenuKeydown(event)}
-                      />
-                    )
-                )}
-              </nav>
-            )
-        )}
+        {this.items
+          ?.filter((item) => this.isActive(item))
+          .map(
+            (item) =>
+              item.navbarItems?.length && (
+                <nav class={{ 'sub-menubar': true, 'shadow-wrapper': true }}>
+                  <ul role="menubar">
+                    {item.navbarItems.map((subitem) => (
+                      <Fragment>
+                        <li role="none">
+                          <a
+                            class={{
+                              'menubar-item': true,
+                              'active': this.isActive(subitem),
+                            }}
+                            href={subitem.href}
+                            id={subitem.id}
+                            role="menuitem"
+                            tabIndex={-1}
+                            aria-haspopup={subitem.menuItems?.length ? 'true' : 'false'}
+                            aria-expanded={
+                              subitem.menuItems?.length ? (this.openMenu === subitem.id ? 'true' : 'false') : undefined
+                            }
+                            aria-current={this.current === item.id ? 'page' : 'false'}
```

Comment:
```suggestion
                            aria-current={this.current.includes(subitem.id) ? 'page' : 'false'}
```

anche nell'altro `aria-current` a riga 473 va fatta la stessa cosa (ovviamente con item al posto di subitem)